modules.d/35network-legacy/module-setup.sh

```diff
@@ -32,6 +32,7 @@ install() {
     inst_script "$moddir/ifup.sh" "/sbin/ifup"
     inst_script "$moddir/dhcp-multi.sh" "/sbin/dhcp-multi.sh"
     inst_script "$moddir/dhclient-script.sh" "/sbin/dhclient-script"
+    inst_script "$moddir/99-default.link" "/usr/lib/systemd/network/99-default.link"
```

Comment:
Hmm, this is missing in the `35network-manager` module also

Reply:
Hmm none of the nm test failed after adding this to legacy but I'll add it and throw a test at test-30 to see if it fails without the include line in the process.